src/interpolations.jl

```diff
@@ -32,6 +29,17 @@ The following interpolations are implemented:
 * `Serendipity{RefQuadrilateral,2}`
 * `Serendipity{RefHexahedron,2}`
 
+The following interpolations are implemented with arbitrary order:
+
+* `Lagrange{RefTriangle,order}`
```

Comment:
I think we should wrap this also into the arbitrary order Lagrange struct.

Reply:
I think the name is kinda too long 😄. Maybe we can make it a fallback for Lagrange constructor so we don't have to use this long name?